### ISSUE TITLE
PrimeFaces Mobile does not support PrimeFaces.scrollTo (Uncaught TypeError)

### ISSUE THREAD

**Issue by cnsgithub** [primefaces/primefaces#2569]
Hi, i noticed that with PrimeFaces mobile there will be an error when trying to use. Is there an additional *.js file to be included manually for this ? (jquery-ui). I'd like to avoid manual additions though as it should work out of the box

```
PrimeFaces.scrollTo("j_idt7:switch")
```
It can be reproduced by using a browser's console on this page:

https://www.primefaces.org/showcase/mobile/forms.xhtml

![image](https://user-images.githubusercontent.com/13161711/28571891-221408f4-7145-11e7-80c6-d911c3979ba9.png)

**Comment by tandraschko** [comment]
A PR would be great.

**Comment by cnsgithub** [comment]
Well it's up to you / the devs to decide if you want to include jquery-ui when on PF mobile or not (or try to avoid using 'easing'. My personal preference would be to keep mobile slim and fast and avoid adding too much dependencies.
---
Possible solution:

JQuery checks for duration, before using easing
![image](https://user-images.githubusercontent.com/13161711/28656329-a5f67848-72a1-11e7-8881-5477a7bc9cde.png)

core.js:
```
        /**
         *  Scrolls to a component with given client id
         */
        scrollTo: function(id) {
            var offset = $(PrimeFaces.escapeClientId(id)).offset();

         // Somehow detect if we are on mobile and don't use EASING. The jquery function checks for "duration" of easing and if it's undefined scrolling would work it seems

            $('html,body').animate({
                scrollTop:offset.top
                ,
                scrollLeft:offset.left
            },{
                easing: 'easeInCirc'
            },1000);

        },
```

---
BTW: I **know** that a PR is always great, but I think it's not necessary to reply with it on every issue. I'm not that much into the PrimeFaces codebase and especiall it's build process with maven + honestly i don't have much time for it as i use PF for work and obviously the priority is to fix bugs on the own codebase. I usually give detailed description and it's **very** easy to reproduce the issues i post (compared to some others). Sorry if that sounds harsh but i have the feeling that 90% of the issues i post are being ignored or trivialized. Sure most of them are not crucial, maybe even slight weaknesses. But I'm starting to feel that it's useless to post problems anyway without a PRO account..

**Comment by tandraschko** [comment]
PrimeFaces Mobile will be deprecated  in 6.2 and removed in 6.3 ( see https://www.primefaces.org/primefaces-6-2-roadmap/)
Therefor i will close this issue as won't fix. If you will provide a PR to fix this issue until 6.2 RC1, i can merge it.
